packages/date-time/src/components/Calendar/docs/CalendarDonts.md

```diff
@@ -1,4 +1,4 @@
 - Don't attempt to break apart year from month/day selectors. If granularity is required, use the Dropdown control or something similar.
 - Don't attempt to force resize the control in any way.
 - Don't force the control to render one mode vs. the other (year or month/day)
-- The flyout selector is a light dismiss control. Don't modify this behavior in any way.
+- Don't modify flyout selector's behavior in any way. The flyout selector is a light dismiss control.
```

Comment:
I have no idea what it's saying. @ecraig12345 ?

recommend removing.

Reply:
Light dismiss means you can click outside it to dismiss, as opposed to only dismissable by clicking the X (or some other close button)

Reply:
It seems to be a pretty standard term in our docs, but the meaning isn't necessarily obvious if you're not familiar with it...

Reply:
What does modify behavior mean exactly though? I don't know how to visualize this.